GestureHandlerModule.web.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { TouchableWithoutFeedback, View } from 'react-native';
+
+import State from './State';
+
+function handleHandlerStateChange(props, event, oldState, state) {
+  const { enabled, onHandlerStateChange } = props;
+
+  if (enabled !== false && onHandlerStateChange) {
+    const {
+      nativeEvent: {
+        locationX: x,
+        locationY: y,
+        pageX: absoluteX,
+        pageY: absoluteY,
+      },
+    } = event;
+
+    onHandlerStateChange({
+      nativeEvent: {
+        oldState,
+        state,
+        x,
+        y,
+        absoluteX,
+        absoluteY,
+        pointerInside: true,
+      },
+    });
+  }
+}
+
+function handleFailed(props, event) {
+  handleHandlerStateChange(props, event, State.ACTIVE, State.FAILED);
+}
+
+function handleEnd(props, event) {
+  handleHandlerStateChange(props, event, State.ACTIVE, State.END);
+}
+
+function handleActivate(props, event) {
+  handleHandlerStateChange(props, event, State.BEGAN, State.ACTIVE);
+}
+
+function handleBegan(props, event) {
+  handleHandlerStateChange(props, event, State.UNDETERMINED, State.BEGAN);
+}
+
+class UnimplementedGestureHandler extends React.Component {
+  render() {
+    return this.props.children;
```

Comment:
do we want to add a warning if an unsupported GH is being rendered on web?

Reply:
Yep, it's implemented now.